v3/lints/cabf_br/lint_cabf_crl_valid_reason_codes.go

```diff
@@ -0,0 +1,73 @@
+package cabf_br
+
+/*
+ * ZLint Copyright 2023 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
+)
+
+type crlHasValidReasonCodes struct{}
+
+func init() {
+	lint.RegisterRevocationListLint(&lint.RevocationListLint{
+		LintMetadata: lint.LintMetadata{
+			Name:          "e_cab_crl_has_valid_reason_code",
+			Description:   "CRL Reason codes 1, 3, 4, 5, 9.",
```

Comment:
```suggestion
			Description:   "Only the following CRLReasons MAY be present: 1, 3, 4, 5, 9.",
```
Aligns a little better from the text in the BRs, in my opinion.